src/SprykerEco/Zed/UnzerGui/Communication/Plugin/MerchantGui/Tabs/MerchantProfileUnzerFormTabExpanderPlugin.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Communication\Plugin\MerchantGui\Tabs;
+
+use Generated\Shared\Transfer\TabsViewTransfer;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\MerchantGuiExtension\Dependency\Plugin\MerchantFormTabExpanderPluginInterface;
+
+/**
+ * @method \SprykerEco\Zed\UnzerGui\Communication\UnzerGuiCommunicationFactory getFactory()
+ */
+class MerchantProfileUnzerFormTabExpanderPlugin extends AbstractPlugin implements MerchantFormTabExpanderPluginInterface
```

Comment:
Plugin names should have a suffix of plugin interface name, 
e. g. `MerchantFormTabExpanderPluginInterface` → `[MerchantProfileUnzer]MerchantFormTabExpanderPlugin` or `[ProfileUnzer]MerchantFormTabExpanderPlugin`

Reply:
Fixed